gen/tests/utils.py

```diff
@@ -0,0 +1,43 @@
+"""
+Utilities for tests for ``gen``.
+"""
+
+import copy
```

Comment:
Please note in review that these utilities are not new. They are instead moved from `test_validation.py`.